mesop/examples/chat.py

```diff
@@ -0,0 +1,40 @@
+import random
+import time
+
+import mesop as me
+import mesop.labs as mel
+
+
+def lorem_ipsum_chat(input):
```

Comment:
nit: I would move this below `def page`; we can use this example and display it in the docs, and it's better to show the main code upfront.

Reply:
Also: add a type annotation for input (mostly for better documentation)